philipmeneghini/InstaConnect
# Interactive Comments

**Summary**
Created a new `Comment.tsx` component which adds hover menu items for comments on posts. Currently it allows you to just like the comment if you did not create the comment. If you created the comment then you can delete or edit the comment.

![comment](https://github.com/philipmeneghini/InstaConnect/assets/63440066/0e1c31d4-1e49-4cf0-865a-ec6681561cb4)

**Testing**

- Make sure that changes including likes, edits and deletions persist to the mongo db in the backend
- Test hover functionality
- Verify that you can click on comment authors and be directed to their profile
